CHANGELOG.md

```diff
@@ -7,7 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.11.0] - 2020-01-26
+## [0.12.0] - 2021-01-27
+
+### Changed
+
+- Bump `dusk-plonk` to `v0.5` [#75](https://github.com/dusk-network/hades252/issues/75)
```

Comment:
I honestly don't think is necessary add the issue for such trivial thing to the changelog. It should be done only for complex topic where the issue provide further details on the matter (otherwise, it should be also duplicated for the lines below since they're all related to the same issue).

Reply:
Fixed